.asf.yaml

```diff
@@ -50,6 +50,74 @@ github:
     main:
       required_pull_request_reviews:
         required_approving_review_count: 1
+    # needs to be updated as part of the release process
+    # Github doesn't support wildcard branch protection rules, only exact branch names
```

Comment:
I believe GitHub does support that. At least I have set wildcard-based protection rules in the past. Might it be that ASF automation does not?

Reply:
Appreciate if you can point me to that doc. What I understood from  https://github.com/apache/infrastructure-asfyaml?tab=readme-ov-file#branch-protection for `asf.yml` it is exact name support